cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -91,6 +91,39 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           });
         });
 
+        if (service === 'news') {
+          it('should render a visible guidance message', () => {
+            cy.window().then(win => {
+              const aresMedia = getBlockData(
+                'video',
+                win.SIMORGH_DATA.pageData,
+              );
+              const longGuidanceWarning =
+                aresMedia.model.blocks[1].model.blocks[0].model.versions[0]
+                  .warnings.long;
+
+              cy.get('div[class^="StyledVideoContainer"]')
+                .eq(0)
+                .within(() => {
+                  // Check for video with guidance message
+                  if (longGuidanceWarning) {
+                    cy.get('div[class^="StyledGuidance"]')
+                      .should('to.have.descendants', 'svg')
```

Comment:
Is there a requirement for this test to also check the component's form? It's less expensive to do this using unit tests, plus we will have to maintain this when Guidance changes and I think it already has - doesn't the latest version of Guidance not have an SVG?

Reply:
Thanks for the update, while @jamesbrumpton  and i were going through this we hadn't checked what had changed for the new version.